docs/setup_eecs281.md

```diff
@@ -116,12 +137,18 @@ Edit these three lines in the Makefile.  Your values might be different, check t
 ```make
 UNIQNAME = not_awdeorio
 # ...
-EXECUTABLE = main
+IDENTIFIER  = not_a_valid_identifier
```

Comment:
What's this for?  I'd like to include a comment.

Reply:
All EECS 281 projects have a "project identifier" which (I believe) is just a hash of the project title. This "project identifier" must be in all .cpp, .c, .h. and .hpp files in a comment, or the AG will throw an error.

The "project identifer" must also be included in the `Makefile`

My understanding is it helps reduce cheating between semesters.

Reply:
It's a SHA-1 digest of the "EECS 281" + project_title.

Reply:
Yes, it will be in the project spec.